mergify_engine/config.py

```diff
@@ -218,9 +218,9 @@ def ApplicationAPIKeys(v: str) -> typing.Dict[str, ApplicationAPIKey]:
             "TESTING_FORWARDER_ENDPOINT",
             default="https://test-forwarder.mergify.io/events",
         ): str,
-        voluptuous.Required("INSTALLATION_ID", default=15398551): voluptuous.Coerce(
-            int
-        ),
+        voluptuous.Required(
+            "TESTING_INSTALLATION_ID", default=15398551
+        ): voluptuous.Coerce(int),
```

Comment:
unrelated?

Reply:
I typed all tests using redis_XXX and some was complaining about missing types, when I added the type for INSTALLATION_ID I prefixed it with TESTING_ as it's testing only thing.